test/parallel/test-fs-write-file-invalid-path.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+if (!common.isWindows)
+  common.skip('This test is for Windows only.');
+
+common.refreshTmpDir();
+
+const DATA_VALUE = 'hello';
+
+// Refs: https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx
+// Ignore '/', '\\' and ':'
+const REVERSED_CHARACTERS = '<>"|?*';
```

Comment:
Unless I am missing something here, I am pretty sure those characters are `RESERVED`, not `REVERSED`.